### ISSUE TITLE
form submit don t work

### ISSUE THREAD

**Issue by brusand** [angular/material#5381]
Hi community,

my first simple material-angular app don't work. A simple login form , and the button type submit never fire.

may i missed something ?

thx
here is my single html page to reproduce my issue (never console log trace)

<!DOCTYPE html>
<html lang="en" ng-app="welcomeApp">
<head>
    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.0.0-rc1/angular-material.min.css">

</head>
<body id="welcomeApp" layout="row">

<div layout="row" flex>
    <div layout="column" flex id="content">
        <md-content layout="column" flex class="md-padding">
            <form name="loginForm" ng-controller="WelcomeController as wc" ng-submit="wc.login()">
                <md-input-container>
                    <label>Username</label>
                    <input type="text" ng-model="wc.email">
                </md-input-container>
                <md-input-container>
                    <label>Password</label>
                    <input type="password" ng-model="wc.password">
                </md-input-container>
                <md-button type="submit" class="md-primary btn btn-primary">Login</md-button>
            </form>
        </md-content>
    </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular-animate.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular-aria.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/angular_material/1.0.0-rc1/angular-material.min.js"></script>

<script>

    var welcomeApp = angular.module('welcomeApp', [
        'ngMaterial'
    ]);

    welcomeApp.controller('WelcomeController', [
    '$scope', function($scope) {
      $scope.login = function() {
          console.log("login = " + $scope.email + " password : " + $scope.password);
      }
    }
]);
</script>

</body>
</html>

**Comment by EladBezalel** [comment]
Can you please post a codepen with your problem so we can investigate it more?

**Comment by brusand** [comment]
Thx
here is a CodePen
http://codepen.io/burno75/pen/RWyGyJ

on submit an alertbox should popup

thx

**Comment by EladBezalel** [comment]
@brusand Seems like something in your `Controller as ws` 
[codepen](http://codepen.io/EladBezalel/pen/jbxVKB)

**Comment by RolandHeath** [comment]
@brusand Elad is right, you're mixing the $scope syntax with controllerAs syntax. 
You can rewrite your app without using $scope at all - you'll probably find some good advice on stackoverflow or similar as to how to do this.

This is not an issue with angular material.
